### ISSUE TITLE
add Manifesto Tracker, Court Cases

### ISSUE THREAD

**Issue by wkambale** [kelvinndmo/DECISION-2027#1]
- the platform can have a section to track manifestos of MPs to show progress.
- add court cases that arise from the reckless bills that they bring to the parliament

**Comment by geekswagg** [comment]
@wkambale do they even have manifestos? Where can they be found?

For the court cases, the cause list is available publicly here: https://efiling.court.go.ke/kiosks. Challenge is, you have to have the case number. We can then log the judgements, hearings, etc

**Comment by wkambale** [comment]
@geekswagg, getting the manifestos might be problematic indeed. i only see party manifestos. getting individual ones would mean tracking their campaign speeches and extracting them.

**Comment by geekswagg** [comment]
> @geekswagg, getting the manifestos might be problematic indeed. i only see party manifestos. getting individual ones would mean tracking their campaign speeches and extracting them.

We can start with their JD maybe?

**Comment by wkambale** [comment]
> > @geekswagg, getting the manifestos might be problematic indeed. i only see party manifestos. getting individual ones would mean tracking their campaign speeches and extracting them.
> 
> We can start with their JD maybe?

perhaps. that could work. also, we could look at their committee roles in parliament.

**Comment by geekswagg** [comment]
> > > @geekswagg, getting the manifestos might be problematic indeed. i only see party manifestos. getting individual ones would mean tracking their campaign speeches and extracting them.
> > 
> > 
> > We can start with their JD maybe?
> 
> perhaps. that could work. also, we could look at their committee roles in parliament.

Qualifies for first PR, no?

**Comment by wkambale** [comment]
> > > > @geekswagg, getting the manifestos might be problematic indeed. i only see party manifestos. getting individual ones would mean tracking their campaign speeches and extracting them.
> > > 
> > > 
> > > We can start with their JD maybe?
> > 
> > 
> > perhaps. that could work. also, we could look at their committee roles in parliament.
> 
> Qualifies for first PR, no?

yeah, i think so. i have setup the environment and good to dive in.
